src/index.ts

```diff
@@ -34,8 +36,13 @@ export * from './utils/terms'
 
 const NextId = BlankNode.nextId
 
-const { fromNT } = Formula.prototype;
 
+// Prepare bound versions of data factory methods for export
+const boundDataFactory = {} as IRDFlibDataFactory
+for (const name in DataFactory) {
+  if (typeof DataFactory[name] === 'function')
+    boundDataFactory[name] = (...args) => DataFactory[name](...args)
```

Comment:
I might be wrong, but I think this way of doing it unnecessarily gives those methods access to lexical scope, so I think
```js
boundDataFactory[name] = boundDataFactory[name].bind(boundDataFactory)
```
would be slightly cleaner, but it would definitely work this way too, so that's probably fine.

Thanks a lot for jumping in to fix this one!

Reply:
They're equivalent, but I seem to remember that anonymous functions a preferred over `bind` in V8 for speed reasons. And it's a `const` so can be propagated.

Reply:
Ah ignore that, actually, the scope wouldn't make it into the `DataFactory[name](...args)` call anyway.

Reply:
(My knowledge is, however, outdated: https://stackoverflow.com/questions/17638305/why-is-bind-slower-than-a-closure/17638540#17638540)

Reply:
Changed to `bind` give no performance difference; saves the `name` lookup as well.